components/calendar/src/cal/hijri.rs

```diff
@@ -129,12 +142,21 @@ pub trait Rules: Clone + Debug + crate::cal::scaffold::UnstableSealed {
 
 /// [`Hijri`] [`Rules`] based on an astronomical simulation for a particular location.
 ///
-/// These simulations are unofficial and are known to not necessarily match sightings
-/// on the ground. Unless you know otherwise for sure, instead of this variant, use
-/// [`UmmAlQura`], which uses the results of KACST's Mecca-based calculations.
+/// These rules use the relative positions of the Earth, moon, and sun to predict the
+/// exact moment of a new moon, and then apply a criterion proposed by S. K. Shaukat in
+/// 1996 to determine crescent moon visibility.
```

Comment:
link?

Reply:
Added a citation

Reply:
linking to an undocumented method is not a citation